File: laramie-administracion-webapi/src/domain/entities/documento.ts
export default class Documento {

    id: number;
	idTipoPersona: number;
	idPersona: number;
	idTipoDocumento: number;
	numeroDocumento: string;
	principal: boolean;

	constructor(
        id: number = 0,
		idTipoPersona: number = 0,
		idPersona: number = 0,
		idTipoDocumento: number = 0,
		numeroDocumento: string = "",
		principal: boolean = false
	)
	{
        this.id = id;
		this.idTipoPersona = idTipoPersona;
		this.idPersona = idPersona;
		this.idTipoDocumento = idTipoDocumento;
		this.numeroDocumento = numeroDocumento;
		this.principal = principal;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.idTipoPersona = row.idTipoPersona ?? 0;
		this.idPersona = row.idPersona ?? 0;
		this.idTipoDocumento = row.idTipoDocumento ?? 0;
		this.numeroDocumento = row.numeroDocumento ?? "";
		this.principal = row.principal ?? false;
	}

}

File: modules/administracion/models/documento.py
from sqlalchemy import Column, BigInteger, String, Boolean

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class Documento(Base):
    __tablename__ = "administracion_documentos"

    id = Column(BigInteger, primary_key=True, autoincrement=True)
    id_tipo_persona = Column(BigInteger, nullable=False)
    id_persona = Column(BigInteger, nullable=False)
    id_tipo_documento = Column(BigInteger, nullable=False)
    numero_documento = Column(String(50), nullable=False)
    principal = Column(Boolean, nullable=False, default=False)
